src/modules/elements/card/style.pcss

```diff
@@ -0,0 +1,22 @@
+.tribe-editor__card {
+	background-color: var(--tec-color-background);
+	border: 1px solid var(--tec-color-border-secondary);
+	border-radius: var(--tec-border-radius-default);
+	display: flex;
+	flex-direction: column;
+	margin-top: var(--tec-spacer-6);
+	max-width: 650px;
+	overflow: hidden;
+	padding: 0px var( --tec-spacer-5 );
+	position: relative;
+	transition: all 0.3s ease;
```

Comment:
There is a custom prop for this `--tec-transition` ...although it's `0.2s` instead of `0.3s`

Reply:
using the `var(--tec-transition)` now